doniclient/osc/cli.py

```diff
@@ -56,7 +56,17 @@ def take_action(self, parsed_args):
             data = hw_client.export()
         else:
             data = hw_client.list()
-
+        
+        worker_types = set([j['worker_type'] for i in data for j in i['workers']])
+        print(worker_types)
+        print(labels)
+        pep = (
+            oscutils.get_dict_properties(
+                    s, columns, formatters={"Properties": oscutils.format_dict}
+            )
+            for s in data
+        )
+        print(i for i in pep)
```

Comment:
These look like debugging commands for the next PR?

Reply:
Ah. right.